api/util/logging-helpers.js

```diff
@@ -9,16 +9,24 @@ const axios = require('axios');
  * @param {(string|undefined)} errorToAdd.errorTime- The time the error occured
  * @param {string} errorToAdd.apiEndpoint - The location of the error
  * @param {string} errorToAdd.errordescription - The description of the error
- * @returns {boolean} - Whether the save was successul or not
+ * @returns {Promise<boolean>} - Whether the save was successul or not
  */
 async function addErrorLog(errorToAdd) {
-  let errorSaved = true;
-  await axios.post(LOGGING_API_URL + '/api/ErrorLog/addErrorLog',
-    { ...errorToAdd })
-    .catch(err => {
-      errorSaved = false;
+  const newError = new ErrorLog({
```

Comment:
this functions in this file are updated to save stuff to mongodb directly, instead of making an http request